int32 number1
int32 number2
---
int32 result